noetic/distribution.yaml

```diff
@@ -579,6 +579,12 @@ repositories:
       url: https://github.com/ros-perception/calibration.git
       version: noetic-devel
     status: maintained
+  cam2lidar:
+    source:
+      type: git
+      url: https://github.com/up2metric/cam2lidar.git
```

Comment:
@anastasispk please, take a look at our [contribution guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-package-contents) and make sure you meet the requirements. 

At the very least, I see that (a) you're lacking a root LICENSE file, (b) your `package.xml` doesn't specify a license, (c) your `package.xml` doesn't specify a version. Moreover, I see lots of boilerplate in there. You may want to polish it a bit.